hack/drone.jsonnet

```diff
@@ -119,6 +119,9 @@ local Pipeline(name, steps=[], depends_on=[], with_buildkit=false, with_docker=t
     if with_docker then docker,
     if with_buildkit then buildkit,
   ],
+  [ if disable_clone then 'clone']: {
```

Comment:
Added this chunk in here since I had a failure to notify on one of the previous builds. This disables the clone step for our notify pipeline.